### PR TITLE
Added the Doc page for IndexToArrayItemConverter.

### PR DESCRIPTION
**Pull request by CliffAgius**

### CODE REVIEW

**Comment by PRMerger10** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

File: docs/maui/converters/index-to-array-item-converter.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+                <x:String>Value 3</x:String>
+                <x:String>Value 4</x:String>
+                <x:String>Value 5</x:String>
+            </x:Array>
+        </ResourceDictionary>
+    </ContentPage.Resources>
+
+    <StackLayout>
+
+        <Label IsVisible="{Binding MyIntegerValue, Converter={StaticResource IndexToArrayItemConverter}, ConverterParameter={StaticResource MyArray}}" />
```
Comment (bijington) [inline docs/maui/converters/index-to-array-item-converter.md:40]: I am not sure binding to `IsVisible` will provide anything sensible here, perhaps we bind to the `Text` property given our array items are strings?

```suggestion
        <Label Text="{Binding MyIntegerValue, Converter={StaticResource IndexToArrayItemConverter}, ConverterParameter={StaticResource MyArray}}" />
```

File: docs/maui/converters/index-to-array-item-converter.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+class IndexToArrayItemConverter : ContentPage
+{
+    public IndexToArrayItemConverter()
+    {
+        var array = new string[] { "Value 1", "Value 2", "Value 3", "Value 4", "Value 5" };
+
+        var label = new Label();
+
+        label.SetBinding(
+            Label.IsVisibleProperty,
```
Comment (bijington) [inline docs/maui/converters/index-to-array-item-converter.md:60]: I am not sure binding to `IsVisible` will provide anything sensible here, perhaps we bind to the `Text` property given our array items are strings?

```suggestion
            Label.TextProperty, 
```

File: docs/maui/converters/index-to-array-item-converter.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+class IndexToArrayItemConverter : ContentPage
+{
+    public IndexToArrayItemConverter()
+    {
+         var array = new string[] { "Value 1", "Value 2", "Value 3", "Value 4", "Value 5" };
+
+        Content = new Label()
+            .Bind(
+                Label.IsVisibleProperty,
```
Comment (bijington) [inline docs/maui/converters/index-to-array-item-converter.md:86]: I am not sure binding to `IsVisible` will provide anything sensible here, perhaps we bind to the `Text` property given our array items are strings?

```suggestion
                Label.TextProperty, 
```

**Review by bijington — CHANGES_REQUESTED** [review]
Thanks for this @CliffAgius, just a minor change to the property we are binding to but the rest looks good to me!

File: docs/maui/converters/index-to-array-item-converter.md (hunk 1 above)
Comment (CliffAgius) [reply docs/maui/converters/index-to-array-item-converter.md:40]: I thought the same as I copied this from the XCT docs and wondered if it was some magic in the Converter but looking at the sample and api your correct so will make the change,

**Comment by PRMerger17** [comment]
## PRMerger Results
 | Issue | Description |
 --- | --- |
 Added File(s) | This PR contains added files. New files require human review.
 Yaml File(s) | This PR includes changes to .yml file(s) owned by another author.
 File Change Percent | This PR contains file(s) with more than 20% file change.
 Index File | This PR contains a change to the index file.

**Comment by CliffAgius** [comment]
Thanks for the catch @bijington need to be careful copying code from the XCT Docs as they are wrong over there. 😖

    https://docs.microsoft.com/en-us/xamarin/community-toolkit/converters/indextoarrayitemconverter

**Comment by bijington** [comment]
Yes sadly the XCT docs are not in the greatest shape which is why the team have tried to come up with this new proposal approach to hopefully help with getting docs for things and then we make sure they make sense. I may need to go over and fill in some missing bits on the XCT side but hopefully as we get these docs up to speed and .NET MAUI hits GA developers might start to consider moving over in the not so distant future.

We really appreciate all this help!

**Review by bijington — APPROVED** [review]
Looks good to me! Sadly I don't have the power to merge but once @jfversluis has a look then this should get merged.

Thanks again!

**Review by jfversluis — APPROVED** [review]

**Pull request merged**